ui/src/components/LoginComp.vue

```diff
@@ -0,0 +1,60 @@
+<template>
```

Comment:
Please, include validation - if empty fields - use red color to indicate which field is empty.
Apply Frontend validation for the Email - it should be valid email, otherwise considered invalid.
Name the field Email, not Login.
In case of wrong input - e.g. invalid combination of username and password - show a label on the form with Red which says invalid username and/or password.